static/sass/_v1_pattern_navigation.scss

```diff
@@ -0,0 +1,181 @@
+    }
+
+    .p-breadcrumbs__item {
+      position: relative;
+      left: 50%;
+      transform: translate(-50%, 0);
+      width: 100vw;
+    }
+
+    .p-breadcrumbs__link {
+      border-bottom: 1px solid $color-mid-light;
+      width: 100%;
+      display: inline-block;
+      padding: $sp-x-small;
+      margin-bottom: $sp-medium;
+      color: $color-dark;
+    }
+
+    .p-inline-list {
+      margin-left: 0;
+    }
+
+    &__menu {
+
+      width: 100%;
+
+      .p-inline-list__item {
+        width: 50%;
+        float: left;
+        margin-right: 0;
+
+        a {
+          padding: $sp-x-small;
+          color: $color-x-dark;
+          display: inline-block;
+        }
+
+        .is-active {
+          font-weight: bold;
+        }
+      }
+    }
+  }
+
+  &__menu {
+    display: inline-block;
+    margin-left: $sp-large;
+    position: relative;
+  }
+
+  //
+  // large screen view
+  //
+  @media (min-width: $breakpoint-medium) {
+    @include full-width-breakout;
+    background: $color-light;
+    padding: $sp-small 0;
+
+    &__menu::before {
+      content: '\203A';
+      font-weight: 400;
+      color: $color-mid-dark;
+      left: -$sp-small;
+      position: absolute;
+      top: 0;
+    }
+
+    a {
+      color: $color-dark;
+    }
+
+    .p-breadcrumbs__link {
+      color: $color-mid-dark;
+    }
+  }
+
+  .row {
+    padding-left: $sp-x-small;
+    padding-right: $sp-x-small;
+  }
+}
+
+// rollover hover dropdown menu
+.p-navigation__link {
+  position: relative;
+}
+
+@media (min-width: $breakpoint-medium) {
+  .p-navigation__link:hover .hover-menu {
+    display:block;
+  }
+
+  .p-navigation .p-navigation__nav ul li:hover ul::after {
+    background: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAICAYAAAD5nd/tAAAABmJLR0QA/wD/AP+gvaeTAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAB3RJTUUH3QYODgYVPPJJpQAAAT9JREFUKM+dkD9IAnEUx7+/IkVxFaxoKYxIWhqKwJw0JAJ315Yac8+1tUFo6HCp34XRdRp11z+J4BzckqShPzdEQ5G/EhPO6V5Tcg3G0QcePHjvffjygB4oXAIApFMxyPlcWM7nwulU7NfMNQqXGABsZNcCCpcKtapBtapBCpcK2fXlAAAcy3vMlUxTdgEAO9ubybPiPjWFoE6nQ5ZlUfNDULl0RAdSPuncddL30+gnCgOAlngLaocyn4nG9fmFJXj9fhARAMDr82MuEcfk7LSuFQu8DTvovO1SuTxlAKCqPFO5Ov/6FIIsy/qzROOdrst6S1V5xunomo0LrT4yOh4JDg6BMXfvIdvGs/mExutLPZpYnAIAdmNUVuwBbI1NRODxePEf2u0WHu9u4ev3rTLTfKBQaNh1qp5piWCa9/gGBheo3r6AmYcAAAAASUVORK5CYII=') no-repeat;
+    content: '';
+    display: block;
+    height: $sp-x-small;
+    left: $sp-x-large;
+    position: absolute;
+    top: -$sp-x-small;
+    width: 200px;
+    z-index: 999;
+  }
+}
+
+.hover-menu {
+  box-shadow: 0 2px 2px -1px #888;
+  border-radius: 10px;
+  background: #f7f7f7;
+  border: 1px solid #cdcdcd;
+  display: none;
+  float: none;
+  margin: 0;
+  padding: $sp-x-small 0 $sp-medium;
+  position: absolute;
+  top: 60px;
+  left: 0;
+  width: 200px;
+  z-index: 1;
+
+  li {
+    padding: $sp-x-small $sp-small 0;
+
+    &:last-child {
+      padding-bottom: 0;
+    }
+  }
+
+  a {
+    color: $color-dark!important;
```

Comment:
Can you comment why we need to important this? Also, add a space before the `!important`.